src/test/java/io/fabric8/maven/docker/assembly/DockerAssemblyManagerTest.java

```diff
@@ -2,22 +2,13 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Date;
+import java.util.*;
 
 import io.fabric8.maven.docker.config.AssemblyConfiguration;
+import io.fabric8.maven.docker.config.AssemblyMode;
 import io.fabric8.maven.docker.config.BuildImageConfiguration;
-import io.fabric8.maven.docker.util.AnsiLogger;
-import io.fabric8.maven.docker.util.DockerFileUtil;
-import io.fabric8.maven.docker.util.Logger;
-import io.fabric8.maven.docker.util.MojoParameters;
-import mockit.Expectations;
-import mockit.Injectable;
-import mockit.Mock;
-import mockit.MockUp;
-import mockit.Tested;
-import mockit.Verifications;
+import io.fabric8.maven.docker.util.*;
+import mockit.*;
```

Comment:
nit, we usually avoid wildcard imports. I think this might be due to IDE configuration.

Reply:
I always forget to go turn that setting back off... I've replaced them with single imports.